app/frontend/wallet/shelley/helpers/addresses.ts

```diff
@@ -69,4 +86,5 @@ export const isByron = (address: HexString): boolean => {
 }
 
 export const addressToHex = (address: string): HexString =>
+  // TODO: this is pretty fragile, we should check for the validity of the address
```

Comment:
not sure if validate here necessarily - we should be able to leverage typescript to narrow the type of the given string to _Address which this function would expect and then this should no longer be as fragile

Reply:
I agree. Thnx.

Reply:
I would clarify in the comment what we think needs to be done (restrict type of the input to _Address) to not be confused into adding validation here